test/dynamo/test_repros.py

```diff
@@ -3659,6 +3659,16 @@ def fn():
 
         make_fn(None)()
 
+    def test_call_finally_opcode_python_3_8(self):
+        def fn():
+            try:
+                return 17
+            finally:
+                return 30  # noqa: SIM107, B012
+
+        result = torch.compile(fn, backend="aot_eager")()
+        self.assertEqual(result, 30)
+
```

Comment:
What happened before this when we did try-finally?

Reply:
It would raise an exception as the `CALL_FINALLY` opcode wasn't implemented. As an example, here is the bytecode generated on Python 3.8 and Python 3.10 for this testcase: https://godbolt.org/z/PW6oe81Y6

Reply:
Please pass in a tensor and add a tensor operation inside the function to ensure dynamo doesn't skip this frame.